packages/ajax/test/index.test.js

```diff
@@ -15,16 +13,6 @@ describe('public interface', () => {
     expect(Ajax).to.exist;
   });
 
-  it('exports an instance of Ajax', () => {
```

Comment:
With my previous comment, this test would stay

Reply:
done